### DISCUSSION TITLE
eliminate white space in qmd file

### DISCUSSION THREAD

**Discussion by hj12o** [quarto-dev/quarto-cli#3141] [Q&A]
Hi,

I am looking to:

(1) Eliminate the 1 or 2 lines of white space between the "Section Title" and the body. 

(2) Eliminate the white space between the last plot and the end of the body. 

Can you advise how to remove the white space under the section title and increase the plot size to fill out the entire body?

Thank you!

here is the qmd file. 

````
---
title: "Penguins, meet Quarto!"
format:
    pdf:
      toc: false
      classoption: landscape
editor: source
geometry:
  - showframe
header-includes:
  - \usepackage{sectsty} 
---

```{r}
#| label: load-packages
#| include: false
#install.packages("palmerpenguins")
#tinytex::install_tinytex()
library(tidyverse)
library(palmerpenguins)
library(ggplot2)
p=ggplot(penguins, aes(x=flipper_length_mm, y=bill_length_mm))+geom_point()+theme(plot.background=element_rect(fill="blue"))
```
\newpage
\DeclareSectionCommand[beforeskip=1pt, afterskip=1pt]{section}

# Section Title
```{r}
#| label: plot-penguins
#| warning: false
#| echo: false
#| layout-ncol: 2
#| layout-nrow: 3
#| fig.height: 2.3
#| fig.width: 6
p
p
p
p
p
p
```
````

**Comment by mcanouil** [comment]
Your question is a LaTeX question, you are more likely to get an answer using the raw LaTeX file (`keep-tex: true`) on StackExchange or StackOverflow.

Also, to format properly Quarto code with code cells, you need to surround the whole code by more backticks than you got in the document, i.e., default is 3, then you need 4.

`````
````
---
title: Title
---

```{r}
1 + 1
```

````
`````

(This example was made with 5 backticks)

**Reply by cscheid** [reply]
(I'm going to go ahead and edit the original post for readability, but I'll keep your response for other readers)

**Comment by baptiste** [comment]
Some subset of these does the trick, I think

```
\setlength{\textfloatsep}{0pt}
\setlength{\intextsep}{0pt}
\renewcommand\floatpagefraction{.9999}
\renewcommand\topfraction{.9999}
\renewcommand\bottomfraction{.9999}
\renewcommand\textfraction{.001}   
\setcounter{totalnumber}{50}
\setcounter{topnumber}{50}
\setcounter{bottomnumber}{50}
```

**Comment by eteitelbaum** [comment]
Try setting `block-headings: false` in the YAML and then using the `titlesec` package as described [here](https://tex.stackexchange.com/questions/53338/reducing-spacing-after-headings?rq=1) and [here](https://github.com/quarto-dev/quarto-cli/issues/2686). 

```
include-in-header: 
        text: |
          \usepackage{titlesec}
          \titlespacing\section{0pt}{12pt plus 4pt minus 2pt}{0pt plus 2pt minus 2pt}
```
